vms/platformvm/state/state.go

```diff
@@ -265,6 +292,10 @@ type state struct {
 
 	currentHeight uint64
 
+	addedBlockIDs map[uint64]ids.ID            // map of height -> blockID
+	blockIDCache  cache.Cacher[uint64, ids.ID] // cache of height -> blockID. If the entry is ids.Empty, it is not in the database
+	blockIDDB     database.Database
+
 	addedBlocks map[ids.ID]blocks.Block            // map of blockID -> Block
```

Comment:
why not call this `verifiedBlocks/verifiedBlockIDs` like other VMs?

Reply:
Which VMs? The AVM uses `addedBlocks` as well: https://github.com/ava-labs/avalanchego/blob/master/vms/avm/states/state.go#L164-L170

The invariant for `AddStatelessBlock` is that it is only called for accepted blocks. Calling them verified would be strange as they've been decided already.